src/language-markdown/clean.js

```diff
@@ -43,10 +43,7 @@ function clean(ast, newObj, parent) {
   }
 
   if (ast.type === "definition" || ast.type === "linkReference") {
-    newObj.label = ast.label
-      .trim()
-      .replace(/[\t\n ]+/g, " ")
-      .toLowerCase();
+    newObj.label = ast.label.trim().replace(/[\t\n ]+/g, " ");
```

Comment:
It seem we print `label` as it is, so the value in AST won't change, this line should be removed.